cdm-core/src/test/java/ucar/nc2/grid/TestGridDataset.java

```diff
@@ -29,9 +29,9 @@ public void testBasics() throws Exception {
     try (GridDataset gds = GridDatasetFactory.openGridDataset(filename, errlog)) {
       assertThat(gds).isNotNull();
       System.out.println("readGridDataset: " + gds.getLocation());
-      assertThat(gds.toString()).startsWith("name = GFS_Puerto_Rico_191km_20090729_0000.nc\n"
-          + "location = ../cdm-core/src/test/data/ncml/fmrc/GFS_Puerto_Rico_191km_20090729_0000.nc\n"
-          + "featureType = GRID");
+      assertThat(gds.toString()).startsWith("name = GFS_Puerto_Rico_191km_20090729_0000.nc" + System.lineSeparator()
```

Comment:
thats painful. maybe change to use String.format(".... %n") ?

Reply:
Yeah, that was fugly...sorry about that! Fixed!